### PR TITLE
Add WVD scalingPlan resource cmdlets

### PR DESCRIPTION
**Pull request by lordjeb**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/DesktopVirtualization/examples/Get-AzWvdScalingPlan.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+### Example 1: {{ Add title here }}
```
Comment (msJinLei) [inline src/DesktopVirtualization/examples/Get-AzWvdScalingPlan.md:1]: Please do not leave the example empty

File: src/DesktopVirtualization/examples/New-AzWvdScalingPlan.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+### Example 1: {{ Add title here }}
+```powershell
+PS C:\> {{ Add code here }}
```
Comment (msJinLei) [inline src/DesktopVirtualization/examples/New-AzWvdScalingPlan.md:3]: Please do not leave the example empty

File: src/DesktopVirtualization/examples/Remove-AzWvdScalingPlan.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+### Example 1: {{ Add title here }}
+```powershell
+PS C:\> {{ Add code here }}
```
Comment (msJinLei) [inline src/DesktopVirtualization/examples/Remove-AzWvdScalingPlan.md:3]: Please do not leave the example empty

File: src/DesktopVirtualization/examples/Update-AzWvdScalingPlan.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+### Example 1: {{ Add title here }}
+```powershell
+PS C:\> {{ Add code here }}
```
Comment (msJinLei) [inline src/DesktopVirtualization/examples/Update-AzWvdScalingPlan.md:3]: Please do not leave the example empty

**Review by msJinLei — CHANGES_REQUESTED** [review]
@lordjeb Please join the Azure organization https://github.com/Azure

**Comment by lordjeb** [comment]
@msJinLei I have joined the Azure org and added the examples.

File: src/DesktopVirtualization/docs/Get-AzWvdScalingPlan.md [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+Get-AzWvdScalingPlan -HostPoolName <String> -ResourceGroupName <String> [-SubscriptionId <String[]>]
+ [-DefaultProfile <PSObject>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Get a scaling plan.
+
+## EXAMPLES
+
+### Example 1: {{ Add title here }}
```
Comment (msJinLei) [inline src/DesktopVirtualization/docs/Get-AzWvdScalingPlan.md:49]: Please regenerate the code to update the docs

File: src/DesktopVirtualization/docs/New-AzWvdScalingPlan.md [hunk 6]
```diff
@@ -0,0 +1,320 @@
...
+ [-Schedule <IScalingSchedule[]>] [-Tag <Hashtable>] [-TimeZone <String>] [-DefaultProfile <PSObject>]
+ [-Confirm] [-WhatIf] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Create or update a scaling plan.
+
+## EXAMPLES
+
+### Example 1: {{ Add title here }}
```
Comment (msJinLei) [inline src/DesktopVirtualization/docs/New-AzWvdScalingPlan.md:28]: Please regenerate the code to update the docs

File: src/DesktopVirtualization/docs/Remove-AzWvdScalingPlan.md [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+ [-Confirm] [-WhatIf] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Remove a scaling plan.
+
+## EXAMPLES
+
+### Example 1: {{ Add title here }}
+```powershell
```
Comment (msJinLei) [inline src/DesktopVirtualization/docs/Remove-AzWvdScalingPlan.md:33]: Please regenerate the code to update the docs

File: src/DesktopVirtualization/docs/Update-AzWvdScalingPlan.md [hunk 8]
```diff
@@ -0,0 +1,346 @@
...
+```
+
+## DESCRIPTION
+Update a scaling plan.
+
+## EXAMPLES
+
+### Example 1: {{ Add title here }}
+```powershell
+PS C:\> {{ Add code here }}
```
Comment (msJinLei) [inline src/DesktopVirtualization/docs/Update-AzWvdScalingPlan.md:39]: Please regenerate the code to update the docs

**Review by msJinLei — CHANGES_REQUESTED** [review]

**Comment by lordjeb** [comment]
@msJinLei Regenerated with examples included.

**Review by msJinLei — APPROVED** [review]

**Pull request merged**